src/martian/util.py

```diff
@@ -22,21 +22,30 @@
 from zope import interface
 
 import martian
+from martian.compat3 import CLASS_TYPES
 from martian.error import GrokError, GrokImportError
 
 def not_unicode_or_ascii(value):
-    if isinstance(value, unicode):
+
+    # python3 compatibility
+    if sys.version_info < (3,) and  isinstance(value, unicode):
```

Comment:
There are two spaces after `and`.

I wonder if a `text_type` alias would make this simpler?